cms/static/js/views/xblock_outline.js

```diff
@@ -322,7 +322,7 @@ define(['jquery', 'underscore', 'gettext', 'js/views/baseview', 'common/js/compo
              */
             handleDuplicateEvent: function(event) {
                 var self = this,
-                    xblockType = XBlockViewUtils.getXBlockType(self.model.get('category'), self.parentView.model, true),
+                    xblockType = XBlockViewUtils.getXBlockType(self.model.get('category'), self.parentView.model),
```

Comment:
@mushtaqak Yes we can use this:
```
var xblockMap = {
    gettext('unit'): 'subsection',
}
```
But we don't need the `true` argument here, so why we used this.
No, I don't think so we can test this with some dummy translation.